src/server/acl/acl_family.cc

```diff
@@ -495,7 +492,10 @@ void AclFamily::GetUser(CmdArgList args, ConnectionContext* cntx) {
   }
   auto& user = registry.find(username)->second;
   std::string status = user.IsActive() ? "on" : "off";
-  auto pass = user.Password();
+  auto pass = PasswordsToString(user.Passwords(), user.HasNopass(), false);
+  if (!pass.empty()) {
+    pass.pop_back();
+  }
```

Comment:
A little bit unusual interface with the leftover space 🙂 But yes, it's only used twice, so not important

Reply:
Soooo, it's used in 3 places and two of them needed the extra space, so I thought this was the minimal set of changes required :rofl: I promise I will clean this, just like the other two places that almost have identical printing (acl list and when we serialize the registry to write it via acl save)